### PR TITLE
Allow the plugin to be loaded

### PR DESCRIPTION
**Pull request by dimbleby**
need symbols defined in sproutletappserver.cpp

### CODE REVIEW

File: greeter_test.mk [hunk 1]
```diff
@@ -14,7 +14,8 @@ TARGET := greeter_as.so
 TARGET_TEST := greeter_test
 
 TARGET_SOURCES := greeterappserver.cpp \
-                  greeterplugin.cpp
+                  greeterplugin.cpp \
+		  sproutletappserver.cpp
```
Comment (bossmc) [inline greeter_test.mk]: Indenting!

File: greeter_as.mk [hunk 2]
```diff
@@ -10,7 +10,8 @@ TARGET := greeter-as.so
 TARGET_TEST := greeter-as.so_test
 
 TARGET_SOURCES := greeterappserver.cpp \
-		  greeterplugin.cpp
+		  greeterplugin.cpp \
+		  sproutletappserver.cpp
```
Comment (bossmc) [inline greeter_as.mk]: Indenting...

**Comment by dimbleby** [comment]
Indentation in fact seems to be fine if you look in the file - something funny going on in the diff.

**Comment by bossmc** [comment]
Tabs vs Spaces (Github uses weird tab length to make this obvious).

**Pull request merged**